technologies/job/aws-step-functions/workflow/instanceActions.js

```diff
@@ -0,0 +1,129 @@
+const AWS = require('aws-sdk');
+const { Response, JobStatus, Log, Stream } = require('@saagie/sdk');
+
+/**
+ * Logic to start the external job instance.
+ * @param {Object} params
+ * @param {Object} params.job - Contains job data including featuresValues.
+ * @param {Object} params.instance - Contains instance data.
+ */
+exports.start = async ({ job, instance }) => {
+  try {
+    console.log('START INSTANCE:', instance);
+    AWS.config.update({credentials: { accessKeyId : job.featuresValues.endpoint.aws_access_key_id, secretAccessKey:  job.featuresValues.endpoint.aws_secret_access_key}});
+    AWS.config.update({region: job.featuresValues.endpoint.region});
+
+    var stepfunctions = new AWS.StepFunctions({apiVersion: '2016-11-23'});
```

Comment:
```suggestion
    const stepfunctions = new AWS.StepFunctions({apiVersion: '2016-11-23'});
```

`var` should be avoided, please use `const` or `let`. Here is a link to compare the different possibilities and why `var` is dangerous.

https://github.com/saagie/sdk/wiki/Best-Practices#var-let-and-const-